code/game/gamemodes/miniantags/tourist/tourist_arrivals.dm

```diff
@@ -0,0 +1,152 @@
+/datum/event/tourist_arrivals
+
+	/// Maximum number of spawns.
+	var/max_spawn = 10
+	/// If the event ran successfully
+	var/sucess_run
+	/// Number of tots spawned in
+	var/tot_number = 0
+	/// Number of players spawned in
+	var/spawned_in = 0
+	/// Number of crew
+	var/crew_count = 0
+	/// Number of antags
+	var/antag_count = 0
+	/// Antag limit defined by crew/10 + 1
+	var/max_antag = 0
+
+
+/datum/event/tourist_arrivals/setup()
+	// Getting a list of players that are logged in and not dead.
+	// Checking if they're an antag or not and defining a 'max_antag' number.
+	for(var/mob/living/player in GLOB.mob_list)
+		if(player.mind && player.stat != DEAD)
+			crew_count++
+			if(!player.mind)
+				continue
+			if(player.mind.special_role)
+				antag_count++
+				continue
+	max_antag = round(crew_count / 10) + 1
```

Comment:
```suggestion
	max_antag = round(crew_count / 10, 1) + 1
```
This will round it towards the highest integer, making it always have at least 1 - up to you if you want to keep the extra max antag